java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -456,4 +457,66 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Object[]> zrankWithScore(String key, String member);
+
+    // TODO add @link to ZMPOP when implemented
+    /**
+     * Blocks the connection until it pops one member-score pair from the first non-empty sorted set
+     * in the provided list of key names.<br>
```

Comment:
Blocks the connection until it pops and returns a member-score pair from the sorted sets stored at the specified <code>keys</code>. The sorted sets are checked in the order. 

This should be enough.